packages/devtools_shared/lib/src/deeplink/deeplink_manager.dart

```diff
@@ -0,0 +1,40 @@
+// Copyright 2021 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:meta/meta.dart';
+import 'package:shelf/shelf.dart' as shelf;
+
+import '../server/server_api.dart';
+
+class DeeplinkManager {
+  static final _buildVariantJsonRegex = RegExp(r'(\[.*\])');
```

Comment:
Can you give an example in dart doc of what the output from the process run looks like and what the result of this regex will be?

Reply:
You could also add tests that test this regex matching based on example process output.